### PR TITLE
en,zh: add doc about recreate TiKV Pod

### LINKED ISSUE #1401: Add procedure about how to delete a TiKV Pod and make it join the cluster as a new store
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

* 1. Delete store with pd-ctl
* 2. Refer to https://docs.pingcap.com/tidb-in-kubernetes/dev/maintain-a-kubernetes-node#maintain-an-irrecoverable-node-in-a-short-term steps 4-6, after that, tikv Pod will join the cluster as a new store

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by KanShiori**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

Close #1401 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/maintain-a-kubernetes-node.md [hunk 1]
```diff
@@ -353,114 +353,162 @@ TiDB 是高可用数据库，可以在部分数据库节点下线的情况下正
...
     ```shell
-    kubectl delete -n ${namespace} pod ${pod_name}
+    kubectl exec -n ${namespace} ${cluster_name}-pd-0 -- watch /pd-ctl store ${store_id}
     ```
 
-8. 观察该 TiKV Pod 是否正常调度到其它节点上：
+    <details>
+    <summary>点击查看期望输出</summary>
+
+    ```json
```
[the remark this answers is not in the document]
Comment (KanShiori) [reply zh/maintain-a-kubernetes-node.md:433]: // 是注释，在 Vscode Preview 里看着很正常，不知道文档支不支持。。

File: zh/maintain-a-kubernetes-node.md [hunk 2]
```diff
@@ -353,114 +353,162 @@ TiDB 是高可用数据库，可以在部分数据库节点下线的情况下正
...
-    查询 Pod 使用的 `PersistentVolumeClaim`：
+1. 参考[迁移 TiKV Region Leader](#迁移-tikv-region-leader) 将 Leader 迁移到其他 Pod。
+
+2. 下线 TiKV Pod。
+
+    > **注意：**
+    >
+    > 下线 TiKV Pod 前，需要保证集群中剩余的 TiKV Pod 数不少于 PD 配置中的 TiKV 数据副本数（配置项：`max-replicas`，默认值 3）。假如不符合该条件，需要先操作扩容 TiKV。
+
+    查看 TiKV Pod 的 `store-id`：
```
[the remark this answers is not in the document]
Comment (KanShiori) [reply zh/maintain-a-kubernetes-node.md:406]: 我加个子序列。
step 3 我感觉不算下线 Pod？

[the remark this answers is not in the document]
Comment (csuzhangxc) [reply zh/maintain-a-kubernetes-node.md]: 是说 step.3 的 ```查询 Pod 使用的 `PersistentVolumeClaim`：``` 和 ```删除该 `PersistentVolumeClaim`：```

File: zh/maintain-a-kubernetes-node.md [hunk 3]
```diff
@@ -353,114 +353,162 @@ TiDB 是高可用数据库，可以在部分数据库节点下线的情况下正
...
+    ```
 
-    ```shell
-    watch pd-ctl store ${ID}
+2. 检查 Region Leader 已经全部被迁移走:
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    kubectl -n ${namespace} get tc ${cluster_name} -ojson | jq ".status.tikv.stores | .[] | select ( .podName == \"${pod_name}\" ) | .leaderCount"
```
[the remark this answers is not in the document]
Comment (KanShiori) [reply zh/maintain-a-kubernetes-node.md:393]: Done

**Review by csuzhangxc — APPROVED** [review]

**Comment by ran-huang** [comment]
@KanShiori PTAL [bdc3995](https://github.com/pingcap/docs-tidb-operator/pull/1725/commits/bdc39953f26ef61752b1e1e885f68ce7c9cc777d)

File: zh/maintain-a-kubernetes-node.md [hunk 4]
```diff
@@ -353,114 +353,168 @@ TiDB 是高可用数据库，可以在部分数据库节点下线的情况下正
...
-3. 参考[迁移 TiKV Region Leader](#迁移-tikv-region-leader) 将 Leader 迁移到其他 Pod。
+3. 参考[重建 TiKV Pod](#重建-tikv-pod) 重建一个新的 TiKV Pod。
 
-4. 下线 TiKV Pod。
+## 迁移 PD Leader
 
-    > **注意：**
-    >
-    > 下线 TiKV Pod 前，需要保证集群中剩余的 TiKV Pod 数不少于 PD 配置中的 TiKV 数据副本数（配置项：`max-replicas`，默认值 3）。假如不符合该条件，需要先操作扩容 TiKV。
+如果 PD Leader Pod 所在节点是要维护的节点，则需要将 PD Leader 先迁移到其他节点上的 Pod。
```
Comment (KanShiori) [inline zh/maintain-a-kubernetes-node.md:360]: 这个应该要执行 step 1 才能知道是不是，估计之所以之前才那么写

File: zh/maintain-a-kubernetes-node.md [hunk 5]
```diff
@@ -353,114 +353,168 @@ TiDB 是高可用数据库，可以在部分数据库节点下线的情况下正
...
-    {{< copyable "shell-regular" >}}
+    其中 `${pod_name}` 是其他节点上的 PD Pod。
 
-    ```shell
-    watch pd-ctl store ${ID}
-    ```
+## 迁移 TiKV Region Leader
 
-6. 解除 TiKV Pod 与节点本地盘的绑定。
+如果 TiKV Region Leader Pod 所在节点是要维护的节点，则需要将 TiKV Region Leader 先迁移到其他节点上的 Pod。
```
Comment (KanShiori) [inline zh/maintain-a-kubernetes-node.md:382]: 我理解正常情况都是每个 TiKV 都有 region leader，感觉句话有点多余

**Comment by KanShiori** [comment]
> @KanShiori PTAL [bdc3995](https://github.com/pingcap/docs-tidb-operator/pull/1725/commits/bdc39953f26ef61752b1e1e885f68ce7c9cc777d)

Rest LGTM

**Comment by ran-huang** [comment]
@KanShiori 中英文都改了下，PTAL

**Comment by KanShiori** [comment]
> @KanShiori 中英文都改了下，PTAL

LGTM

**Comment by KanShiori** [comment]
@ran-huang 感觉可以 merge 了 👀

**Review by ran-huang — APPROVED** [review]

**Pull request merged**